src/main/java/dbpath/DBPathConnectionHandler.java

```diff
@@ -60,4 +61,16 @@ public void closeConnection() {
         }
         connection = null;
     }
+
+    public void deleteDatabaseFolder() {
+        SqlSandboxUtils.deleteDatabaseFolder(dbPath.getDatabaseFile());
+    }
+
+    public boolean createDatabaseFolder() {
+        return new File(dbPath.getDatabaseFile()).getParentFile().mkdirs();
+    }
+
+    public String getDatabaseFileForLoggingPurposes() {
```

Comment:
Yeah exactly. In the end I changed it minorly to `getDatabaseFileForDebugPurposes` which loosely includes both logging/printing out for debugging as well as for use in tests, some of which want to be able to test at the system level that a file is actually there or not.